packages/truffle-deployer/test/errors.js

```diff
@@ -66,12 +66,17 @@ describe("Error cases", function() {
         Abstract,
         Loops
       ],
+      networks: {
+        test: {
+          // TODO:
```

Comment:
(TODO what?)

Reply:
It looks like I forgot to remove the TODO; it's no longer necessary to add more details. I put the colon there as it highlights it in my vscode. I'll remove the todo